packages/bpk-docs/src/pages/ThemingPage/ThemingPage.js

```diff
@@ -30,41 +31,78 @@ import {
   NATIVE_HORIZONTAL_NAV,
   NATIVE_SPINNER,
   NATIVE_SWITCH,
+  SPINNERS,
+  TYPOGRAPHY,
+  HORIZONTAL_NAV,
 } from './../../constants/routes';
 
-const NativeTextPage = () => (<DocsPageBuilder
+const CONTEXT_API_URL = 'https://reactjs.org/docs/context.html';
+const CSS_VARIABLES_URL = 'https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables';
+
+const components = [
+  {
+    id: 'native',
+    title: 'Native',
+    blurb: [
+      <Heading level="h3">Themeable components</Heading>,
+      <BpkList>
+        <BpkListItem>
+          <BpkLink href={NATIVE_BUTTON}>Button</BpkLink>
+        </BpkListItem>
+        <BpkListItem>
+          <BpkLink href={NATIVE_HORIZONTAL_NAV}>Horizontal navigation</BpkLink>
+        </BpkListItem>
+        <BpkListItem>
+          <BpkLink href={NATIVE_SPINNER}>Spinner</BpkLink>
+        </BpkListItem>
+        <BpkListItem>
+          <BpkLink href={NATIVE_SWITCH}>Switch</BpkLink>
+        </BpkListItem>
+      </BpkList>,
+    ],
+    readme: nativeReadme,
+  },
+  {
+    id: 'web',
+    title: 'Web',
+    blurb: [
+      <Heading level="h3">Themeable components</Heading>,
```

Comment:
Can we replace these level 3 headings with a paragraph of text, something along the lines of:

> "The following React Native components are themable:"

> "The following web components are themable:"

Reply:
Done